integrations/SD_Lon/sd_importer.py

```diff
@@ -362,16 +360,19 @@ def add_people(self):
                     date_from=None
                 )
 
-    def create_ou_tree(self):
+    def create_ou_tree(self, create_orphan_container):
         """ Read all department levels from SD """
-        self.importer.add_organisation_unit(
+        if create_orphan_container:
+            self._add_klasse('Orphan', 'Virtuel Enhed', 'org_unit_type')
+            self.importer.add_organisation_unit(
```

Comment:
Jeg er glad for at de forældreløse puttes i en container, jeg læste nemlig længere oppe, at de alle blev lagret på topniveau med forventning om at blive flyttet - og der tænkte jeg: Det kan ikke være sundt  - istedet skulle man - og det har du så gjort. Muy Bien.

Reply:
👍